README.md

```diff
@@ -116,12 +103,80 @@ require'nvim-treesitter.configs'.setup {
 EOF
 ```
 
-Check [`:h nvim-treesitter-modules`](doc/nvim-treesitter.txt)
-for a list of available modules and its options.
+Each module can also be enabled or disabled interactively through the following commands:
+
+```vim
+:TSBufEnable {module} " enable module on current buffer
+:TSBufDisable {module} " disable module on current buffer
+:TSEnableAll {module} [{ft}] " enable module on every buffer. If filetype is specified, enable only for this filetype.
+:TSDisableAll {module} [{ft}] " disable module on every buffer. If filetype is specified, disable only for this filetype.
+:TSModuleInfo [{module}] " list information about modules state for each filetype
+```
+
+Check [`:h nvim-treesitter-commands`](doc/nvim-treesitter.txt) for a list of all available commands. It may be necessary to reload the buffer (e.g., via `:e`) after enabling a module interactively.
+
+# Supported languages
+
+For `nvim-treesitter` to support a specific feature for a specific language requires both a parser for that language and an appropriate language-specific query file for that feature.
+
+The following is a list of languages for which a parser can be installed through `:TSInstall`; a checked box means that `nvim-treesitter` also contains queries at least for the `highlight` module.
+
+We are looking for maintainers to add more parsers and to write query files for their languages.
+
+<!--This section of the README is automatically updated by a CI job-->
+<!--parserinfo-->
+
```

Comment:
If you're wondering why the README CI fails... Don't worry. I fixed it in another PR.